resources/i18n/ja_JP/cura.po

```diff
@@ -44,113 +44,114 @@ msgstr "%1m"
 
 msgctxt "@action:inmenu menubar:printer"
 msgid "&Add Printer..."
-msgstr "＆プリンターを追加する..."
+msgstr "プリンターを追加(&A)..."
```

Comment:
I love the idea of restoring the `&A`, but just for my curiosity, is it useable at all with a Japanese keyboard ?

Reply:
Yes, it's the traditional style for shortcuts in Japanese applications.
![image](https://github.com/user-attachments/assets/98045023-9cc1-41aa-ae7c-88dd48beb734)

Of course, I have confirmed it works in 5.9.1 like `Alt+a`.

Reply:
Ok nice, thanks for the explanation :slightly_smiling_face: